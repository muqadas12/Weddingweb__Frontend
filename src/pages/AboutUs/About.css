.about-us-img {
  margin-left: 450px;
}

.first-heading-about-us {
  font-family: "Times New Roman", Times, serif;
  text-align: center;
  font-weight: bold;
  font-size: 30px;
}

.second-heading-about-us {
  font-family: "Times New Roman", Times, serif;
  text-align: center;
  font-weight: bold;
  font-size: 17px;
}

.description-about-us {
  text-align: center;
}

.btn-color-change {
  border: "none";
  background-color: "black";
  color: "white";
  margin-left: "170px";
}

.bgg {
  width: "205vh";
  height: "200vh";
}/*# sourceMappingURL=About.css.map */