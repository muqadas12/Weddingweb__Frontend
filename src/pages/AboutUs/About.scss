.btn-color-change-custom-hook {
  margin-left: 220px;
}
.about-us-img {
  margin-left: 450px;
}

.first-heading-about-us {
  font-family: 'Times New Roman', Times, serif;
  text-align: center;
  font-weight: bold;
  font-size: 30px;
}
.second-heading-about-us {
  font-family: 'Times New Roman', Times, serif;
  text-align: center;
  font-weight: bold;
  font-size: 17px;
}
.description-about-us {
  text-align: center;
}
.btn-color-change-custom-hook {
  border: 'none';
  background-color: 'black';
  color: 'white';
  margin-left: '450px';
}
.bgg {
  width: '205vh';
  height: '200vh';
}
