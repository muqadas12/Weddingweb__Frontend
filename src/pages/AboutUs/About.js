/* eslint-disable no-unused-vars */
import React from 'react';
import { Button } from 'antd';
import useRandomColor from '../../components/Hooks/useRandomColor';
import aboutUsImg from '../../Assets/images/about.png';
import './About.scss';

function About() {
  const { color, changeColor } = useRandomColor();

  return (
    <div
      className="bgg"
      style={{ width: '205vh', height: '100vh', backgroundColor: `#${color}` }}
    >
      <img className="about-us-img" src={aboutUsImg} alt="aboutus" />
      <Button
        className="btn-color-change-custom-hook"
        type="button"
        onClick={changeColor}
      >
        Change Bg Color
      </Button>
      <p className="first-heading-about-us">Welcome to Wanclouds Weds</p>
      <p className="second-heading-about-us">
        We are glad and greatful that you are here
      </p>

      <p className="description-about-us">
        Wanclouds weds is platform that allow you to make your dreams true of a
        perfect wedding.
      </p>
      <p className="description-about-us">
        It help you providing the facility to book wedding hall,Car rental,
        Catering, Saloon and Photography.We deal with best dealers to provide
        you quality based services
      </p>
    </div>
  );
}

export default About;
