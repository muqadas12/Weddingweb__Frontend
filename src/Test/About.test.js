/* eslint-disable no-console */
/* eslint-disable prefer-const */
/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import About from '../pages/aboutUs/About';

describe('Testing about us page', () => {
  test('abut', () => {
    let wrapper = shallow(<About />);
    expect(wrapper.exists('.first-heading-about-us')).toEqual(true);
    expect(wrapper).toMatchSnapshot();
  });
});
