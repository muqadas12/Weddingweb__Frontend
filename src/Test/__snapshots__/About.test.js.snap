// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing about us page abut 1`] = `
<div
  className="bgg"
  style={
    Object {
      "backgroundColor": "#undefined",
      "height": "100vh",
      "width": "205vh",
    }
  }
>
  <img
    alt="aboutus"
    className="about-us-img"
    src="about.png"
  />
  <button
    className="btn-color-change"
    onClick={[Function]}
    type="button"
  >
    Change Theme
  </button>
  <p
    className="first-heading-about-us"
  >
    Welcome to Wanclouds Weds
  </p>
  <p
    className="second-heading-about-us"
  >
    We are glad and greatful that you are here
  </p>
  <p
    className="description-about-us"
  >
    Wanclouds weds is platform that allow you to make your dreams true of a perfect wedding.
  </p>
  <p
    className="description-about-us"
  >
    It help you providing the facility to book wedding hall,Car rental, Catering, Saloon and Photography.We deal with best dealers to provide you quality based services
  </p>
</div>
`;
